projects/storefrontlib/src/cms-components/cart/cart-shared/cart-item/cart-item.component.html

```diff
@@ -130,10 +130,8 @@
 
       <!-- Promotion -->
 
-      <ng-container
-        *ngIf="appliedProductPromotions$ | async as appliedProductPromotions"
-      >
-        <cx-promotions [promotions]="appliedProductPromotions"></cx-promotions>
+      <ng-container *ngIf="item.promotions">
```

Comment:
This `*ngIf`, even this `<ng-container>` can be removed, right?

Reply:
You are right, the *ngIf oonly tested if there wa an observable instance I believe.